README.md

```diff
@@ -78,39 +139,73 @@ Props that can be passed to the component are listed below:
 		</tr>
 		<tr>
 			<td><code><b>showCount?:</b> boolean</code></td>
-			<td>Boolean to enable and disable showing count on the chart.</td>
+			<td>Boolean to enable and disable showing count on the bar in summary section.</td>
 			<td><code>true</code></td>
 		</tr>
 		<tr>
 			<td><code><b>showAnimation?:</b> boolean</code></td>
-			<td>Boolean to enable and disable showing animations and transitions on the chart.</td>
+			<td>Boolean to enable and disable showing animations and transitions on the bars in chart.</td>
 			<td><code>true</code></td>
 		</tr>
 		<tr>
 			<td><code><b>styles?:</b> object</code></td>
-			<td>Provides you with a bunch of callback functions to override the default styles.(refer 
+			<td>Provides you with a bunch of style objects and callback functions to override the default styles.(refer 
 			<a href="#style-customizations">Style Customizations</a>)
 			<td><code>undefined</code></td>
 		</tr>
 		<tr>
-			<td><code><b>onChartClick?:</b> (ratingId: string): void</code></td>
-			<td>Click handler for each ratings chart</td>
+			<td><code><b>onBarClick?:</b> (ratingId: string): void</code></td>
+			<td>Click handler for each rating bar in chart</td>
+			<td><code>undefined</code></td>
+		</tr>
+		<tr>
+			<td><code><b>ratingRanks?:</b> object</code></td>
+			<td>An object where each key is a rating id, and the corresponding value is the  rank or weightage in number associated with that rating id, utilized in the computation of the average of ratings, by considering the importance of each rating.</td>
```

Comment:
Let's break this into more sentences. Trying to fit into a single sentence is confusing.